### PR TITLE
TC to perform a restore from a restored backup

### PR DESCRIPTION
**Pull request by sabrarhussaini**
Modified the existing testcase (ScheduleBackupCreationAllNS) in order to include the steps for the automation of below mentioned testcase :
TestRail link: https://portworx.testrail.net/index.php?/cases/view/86270
Task: https://portworx.atlassian.net/browse/PB-4763

**Which issue(s) this PR fixes**
Closes #PB-4763

Special notes for your reviewer:
Vanilla-S3:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3781/

Vanilla-NFS:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3783/

### CODE REVIEW

**Review by ak-px — APPROVED** [review]
LGTM

File: tests/backup/backup_restore_basic_test.go [hunk 1]
```diff
@@ -595,16 +607,96 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
...
+			backupNames = append(backupNames, manualBackupName)
+		})
+
+		Step(fmt.Sprintf("Perform default restore of applications by replacing the existing resources"), func() {
+			log.InfoD(fmt.Sprintf("Perform default restore of applications by replacing the existing resources"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			defaultRestoreName = fmt.Sprintf("%s-%s", restoreNamePrefix, manualBackupName)
+			err = CreateRestoreWithReplacePolicy(defaultRestoreName, manualBackupName, make(map[string]string), SourceClusterName, orgID, ctx, make(map[string]string), 2)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating default restore for manual backup with replace policy [%s]", defaultRestoreName))
```
Comment (snigdha-px) [inline tests/backup/backup_restore_basic_test.go:644]: Here can we separately validate the restore using "ValidateRestore"
Refer : Testcase : KubevirtVMBackupRestoreWithDifferentStates

**Review by snigdha-px — CHANGES_REQUESTED** [review]

File: tests/backup/backup_restore_basic_test.go (hunk 1 above)
Comment (sabrarhussaini) [reply tests/backup/backup_restore_basic_test.go:645]: Added the restore verification.

**Review by snigdha-px — APPROVED** [review]
LGTM

File: tests/backup/backup_restore_basic_test.go [hunk 2]
```diff
@@ -510,12 +511,21 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
...
 		clusterStatus           api.ClusterInfo_StatusInfo_Status
+		srcClusterUid           string
+		multipleRestoreMapping  map[string]string
+		customRestoreName       string
+		backupNames             []string
+		restoreNames            []string
 	)
-	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58015
+
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58015 and 86270(testcase scrubbed)
```
Comment (kshithijiyer-px) [inline tests/backup/backup_restore_basic_test.go:528]: You need not call out 86270 here.

**Review by kshithijiyer-px — APPROVED** [review]
One minor comment fix rest looks good.

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**